src/__Private/get_host_os.hack

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST\__Private;
+
+use namespace HH\Lib\Str;
+
+<<__Memoize>>
+function get_host_os(): ?OperatingSystem {
+    $os = \PHP_OS;
```

Comment:
2 space indentation

Reply:
Vscode prefers `editor.tabSize` over hackfmt. I added the directive to .vscode/setting.json and corrected the indentation.